pom.xml

```diff
@@ -248,12 +249,13 @@
                 <plugin>
                     <groupId>net.revelc.code</groupId>
                     <artifactId>impsort-maven-plugin</artifactId>
-                    <version>1.7.0</version>
+                    <version>1.8.0</version>
                     <configuration>
                         <groups>java.,javax.,org.,com.,io.,io.kroxylicious</groups>
                         <removeUnused>true</removeUnused>
                         <staticAfter>true</staticAfter>
-                        <compliance>${java.test.version}</compliance>
+                        <!-- use langauge level 17, see https://github.com/javaparser/javaparser/issues/3833 -->
+                        <compliance>17</compliance>
```

Comment:
Hmmm That's going to get funky.  Especially as Java 21 has some nice [perf improvements ](https://minborgsjavapot.blogspot.com/2023/01/java-21-performance-improvements.html)coming.

Is impsort valuable enough to us to keep?

Reply:
There's https://github.com/javaparser/javaparser/issues/3907 raised to request Java 19 support.  If this becomes an issue before Java 19 support appears (no harm in waiting), we can either find an alternative import sorter or use something like checkstyle to enforce the same effect.